op-service/eth/supervisor_status.go

```diff
@@ -0,0 +1,17 @@
+package eth
+
+type SupervisorStatus struct {
+	// MinSyncedL1 is the highest L1 block that has been processed by all supervisor nodes.
+	// This is not the same as the latest L1 block known to the supervisor,
+	// but rather the L1 block view of the supervisor nodes.
+	// This may not be fully derived into the L2 data of a particular node yet.
+	// The local-safe L2 blocks were produced/included fully from the L1 chain up to _but excluding_ this L1 block.
```

Comment:
I'm not sure this last bit is right. Since this L1 block may have been partially processed, local-safe blocks _might_ have been added based on it.  That's fine for the challenger since it checks which L1 blocks things were actually derived from separately, but worth making sure the comment is accurate.

Reply:
removed the misleading line. The remaining comment should be descriptive enough.